src/plugins/dev_tools/public/global_search/search_devtool_command.tsx

```diff
@@ -33,12 +33,18 @@ export const searchForDevTools = async (
       <EuiFlexItem>
         <EuiIcon type="consoleApp" color="text" />
       </EuiFlexItem>
-      <EuiFlexItem>{props.title}</EuiFlexItem>
+      <EuiFlexItem>
+        <EuiHighlight search={query}>{props.title}</EuiHighlight>
+      </EuiFlexItem>
     </EuiFlexGroup>
   );
 
   return tools
-    .filter((tool) => tool.title.toLowerCase().includes(query.toLowerCase()))
+    .filter(
+      (tool) =>
+        tool.title.toLowerCase().includes(query.toLowerCase()) ||
+        props.title.toLowerCase().includes(query.toLowerCase())
+    )
```

Comment:
Nit: Seems `props.title.toLowerCase().includes(query.toLowerCase())` has nothing to do with the tools filter, would this look a bit more reasonable?

```suggestion
    const titleMatched = props.title.toLowerCase().includes(query.toLowerCase())
    const displayedTools = titleMatched ? tools : tools.filter((tool) => tool.title.toLowerCase().includes(query.toLowerCase()))
    displayedTools.map(...)
```

Reply:
thanks, it's more clear.